lib/generator/app_localized_class_generator.dart

```diff
@@ -26,7 +26,9 @@ class Localized {
   static const delegate = LocalizedDelegate();
 
   static Localized of(BuildContext context) {
-    return Localizations.of<Localized>(context, Localized);
+    final localized = Localizations.of<Localized>(context, Localized);
+    if(localized == null) throw Exception('Could not find a Localization with the given context.');
```

Comment:
Since Localizations.of(context, Type) returns T? we cannot directly return the value. I think we shouldn't directly expose a null-safe syntax here since a client may have not been opted-in their app to null-safety. @shana0440